README.md

```diff
@@ -114,7 +114,11 @@ Composer shipped as jar, to run it you need JVM 1.8+: `java -jar composer-latest
   * Default: `120` seconds (2 minutes). 
   * Applicable to both test APK and APK under test. 
   * Example: `--install-timeout 20`
-
+* `--fail-if-no-tests`
+  * Either `true` or `false` to enable/disable error on empty test suite.
+  * Default: `true`
+  * Example: `--fail-if-no-tests true`
```

Comment:
nit: Let's specify `false` in example, I try to showcase how to use non-default variant typically because people rarely specify default :)